internal/provider/resource_cloud_router_quick_connect_test.go

```diff
@@ -0,0 +1,37 @@
+package provider
+
+import (
+	"testing"
+
+	"github.com/PacketFabric/terraform-provider-packetfabric/internal/testutil"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccCloudRouterQuickConnectRequiredFields(t *testing.T) {
+	testutil.PreCheck(t, []string{
+		"PF_AWS_ACCOUNT_ID",
+		"PF_QUICK_CONNECT_SERVICE_UUID",
```

Comment:
In order to avoid to add another env variable, let's add a new function to randomly select a quick connect service from the available list.

GET [/v2/marketplace/services](https://docs.packetfabric.com/api/v2/swagger/#/Marketplace/marketplace_customer_services_get)

Filter on `service_type: cloud-router-service`